### ISSUE TITLE
RELU Layer after Last Batch Normalization

### ISSUE THREAD

**Issue by geophysicsQC** [zongyi-li/fourier_neural_operator#26]
https://github.com/zongyi-li/fourier_neural_operator/blob/3daa84926127f39e8e45c8a952d1e20ad19e8780/fourier_2d_time.py#L129-L139

Hi Zongyi,

Should the RELU layer being added after the last batch normalization (line 133 above) referring to your equation 
![image](https://user-images.githubusercontent.com/24968921/105931322-95914180-6010-11eb-9503-208bbc7d3164.png)

Thanks,

Chen

**Comment by zongyi-li** [comment]
Hi Chen,

Thanks for pointing it out. Usually, we don't use the activation layer in the end. To be consistent, you can either add a ReLU layer or delete the batchnorm layer at line 132. In practice, we observe it doesn't influence the result much.

Thanks,
Zongyi

**Comment by geophysicsQC** [comment]
Hi Zongyi,

Thanks very much for instructions! Yes, I have tried Relu layer at line 132 and to be honest the result after the same number of epochs is worse a little bit for my acoustic wave equation problem. This may due to the fact that the wavefield should be having both positive and negative values and the Relu mutes the negative so the FC layers after it has to contain more information to add those back. This also reminds me to try other activation layers for the Green's function part such as sigmoid.

Have a nice day!

Chen Qi
